organizing_hub/signals.py

```diff
@@ -91,6 +94,31 @@ def sync_group_leader_affiliation_for_user(user):
         )
 
 
+@receiver(post_save, sender=ContactList)
+def contact_list_post_save_handler(instance, **kwargs):
+    """
+    Contact List post-save handler
+
+    Start task to send approved event promotion if contact list is complete and
+    list is not empty
+    """
+
+    contact_list = instance
+    if contact_list.status == ContactListStatus.complete.value[0] and (
+        contact_list.contacts.count() > 0
+    ):
+
+        """Check if contact list is attached to approved event promotion"""
```

Comment:
Do we check anywhere that an event has been approved?

Reply:
Separate ticket

Reply:
👍